index.js

```diff
@@ -61,23 +63,27 @@ class Client_AuroraDataMySQL extends Client_MySQL { // eslint-disable-line camel
   _driver () {
     let RDSDataService;
     try {
-      RDSDataService = require('aws-sdk/clients/rdsdataservice');
+      RDSDataService = RDSDataClient;
     } catch (err) { /* istanbul ignore next */
       throw new Error(`Failed to load aws-sdk rdsdataservice client, did you forget to install it as a dependency? (${err.message})`);
     }
 
-    const https = this.config.connection.sdkConfig && String(this.config.connection.sdkConfig.endpoint).startsWith('http:')
+    const isHttp = this.config.connection.sdkConfig && String(this.config.connection.sdkConfig.endpoint).startsWith('http:');
```

Comment:
As minor PR feedback, I suggest not renaming existing variables where their semantic meaning is left unchanged. It can make it difficult to understand changes through git history when one is trying to debug problems and needing to read older commits to understand how behavior changes over time. I’m fine with this change here, though. It’s impact is minor.